src/sentry/analytics/events/alert_created.py

```diff
@@ -10,6 +10,7 @@ class AlertCreatedEvent(analytics.Event):
         analytics.Attribute("organization_id"),
         analytics.Attribute("rule_id"),
         analytics.Attribute("rule_type"),
+        analytics.Attribute("duplicate_rule"),
```

Comment:
making this not required might be easier for those other cases

Reply:
👍